Controllers/UsersController.cs

```diff
@@ -22,7 +22,7 @@ public UsersController(IServicerRepository repository, IMapper mapper)
             _repository = repository;
             _mapper = mapper;
         }
-
+        [AllowAnonymous]
```

Comment:
@mkorzecki Jak coś testujesz, to musisz patrzeć żeby nie commitować zmian, których nie powinno być,